src/orquestra/sdk/_base/_ast.py

```diff
@@ -114,9 +114,7 @@ def visit_Dict(self, node):
             # having a single `None` output.
             self.outputs.add(_AstReturnMetadata(is_subscriptable=False, n_outputs=1))
         else:
-            self.outputs.add(
-                _AstReturnMetadata(is_subscriptable=True, n_outputs=n_elements)
-            )
+            self.outputs.add(_AstReturnMetadata(is_subscriptable=False, n_outputs=1))
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Can you add a comment explaining why we're doing this?
Both sides of the if statement are the same, which may look odd without any explanation.

Reply:
Imma just colapse them into one statement for now